lib/ReactViews/Search/Breadcrumbs.jsx

```diff
@@ -53,47 +53,61 @@ class Breadcrumbs extends React.Component {
       // Note: should it reset the text if a person deletes current search and starts a new search?
       <Box
         left
-        styledHeight={"32px"}
+        // styledHeight={"32px"}
+        styledMinHeight={"32px"}
```

Comment:
Great idea! I like the use of min height so the height is adjusted when it wraps to the next line - probs just remove the commented out `styledHeight`?